### DISCUSSION TITLE
AAC decoding doubts

### DISCUSSION THREAD

**Discussion by ixuhan** [pschatzmann/arduino-libhelix#9] [Q&A] [answered]
Hello, I want to ask you a few questions (I don't know much about audio decoding):
1. AAC decoding does not seem to be effective for M4A? (Do I need to manually unpack the M4A to extract the original AAC data and then use the decoding library for decoding)
2. Does the Helix library have a source code repository? I tried to find the source code on the Internet, but never found the original address?
thank you and best regards :)

**Comment by pschatzmann** [comment, marked as answer]
Just discovered that these questions have never been answerd:

1) libheix does not support any MP4/M4A parsing. You can use the MP4Parser class from the AudioTools library to extract the aac packets. Or you can use another codec e.g. FDK-AAC or FAAD2

2) The original Helix Community website that hosted the source was: http://www.helixcommunity.org That site is no longer active, but the Helix DNA project and its licensing are documented in historical references.
